frontend-implicaction/src/app/forums/components/custom-table-with-search-bar/custom-table-with-search-bar.component.ts

```diff
@@ -18,10 +18,12 @@ export class CustomTableWithSearchBarComponent
   extends BaseWithPaginationAndFilterComponent<Group, Criteria>
   implements OnInit
 {
+  @Input() tableType: 'Posts' | 'Forums' = 'Posts';
+  @Input() labels: string[] = ['Likes', 'Commentaires', 'Vues'];
+
   readonly ROWS_PER_PAGE_OPTIONS = [5];
   isLoading = true;
   title: string;
-  labels: string[];
   posts: Post[];
   searchValue: string = '';
```

Comment:
Vous disposez d'un objet this.criteria.keyword dans la classe mère, autant l'utiliser.